### DISCUSSION TITLE
Updated YouTube Player Controls in Content Player

### DISCUSSION THREAD

**Discussion by rushi-tekdi** [Sunbird-Knowlg/Community#228] [Q&A]
The YouTube player embedded in the content player lacks the latest control features available in the YouTube app, such as fast-forwarding. We would like to know if the current YouTube player can be updated to include these controls.

If this compatibility is available, please provide the steps and API requirements necessary to integrate the latest YouTube player controls.

GitHub Repository:
    [Sunbird Content Player](https://github.com/Sunbird-Knowlg/sunbird-content-player/)

@coolbung @pallakartheekreddy @snehal0904

**Comment by pallakartheekreddy** [comment]
@rushi-tekdi will check and update on this
